### PR TITLE
Add VivoBook X510UA support

### PR DESCRIPTION
**Pull request by agura-lex**
**Description**
Tested the script on the VivoBook X510UA and adjusted fan curve values to what I consider comfortable enough.

### CODE REVIEW

**Comment by agura-lex** [comment]
Sorry I haven't created an issue, I've read contributing guidelines only after forking and pushing my changes. ¯\_(ツ)_/¯

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ It appears that this project could work with a **whole range of ASUS notebook mo
 * ASUS ZenBook UX410UA ([Fernando Sanz](https://github.com/fsanzdev) + [Petr](https://github.com/xtelcz))
 * ASUS ZenBook Flip UX360UAK ([Aleh Filipovich](https://github.com/afilipovich))
 * ASUS VivoBook 15 X512FA ([MartinMyr](https://github.com/MartinMyr))
+* ASUS VivoBook 15 X510UA ([Alexander Agura](https://github.com/agura-lex))
```
Comment (dominiksalvet) [inline README.md]: Please, move your model at the end of the list, or sort the entire list to keep it consistent.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 The changes not yet present in any release are listed in this section.
 
+### Added
+
+* Support for ASUS VivoBook 15 X510UA
```
Comment (dominiksalvet) [inline CHANGELOG.md]: Please, make it consistent with other new model items in the changelog (link, dot).

**Comment by dominiksalvet** [comment]
I wonder why my GitHub Actions CI configuration does not run on pull requests... I will try to fix it.

**Comment by dominiksalvet** [comment]
Hopefully it is fixed in c49276653244fac79c95496986b4e0b8e0ca0096. Please, update your fork with this change... :smile:

**Comment by agura-lex** [comment]
Should be good now

**Comment by dominiksalvet** [comment]
Thank you for the contribution! :heart:

**Pull request merged**